app/graphql/types/sample_filter_type.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module Types
+  # Sample Filter Type
+  class SampleFilterType < BaseInputObject # rubocop:disable GraphQL/ObjectDescription
+    Sample.ransackable_attributes.each do |attr|
```

Comment:
**nitpick (blocking):** Can you change this to `Sample.ransackable_attributes.excluding("id")` since we don't expose the `id` to the user.

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/723/commits/b1d64665f1e06d44daed9b802c6b8592c5f12068